### PR TITLE
Update ca.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**

### CODE REVIEW

File: streams/ca.m3u [hunk 1]
```diff
@@ -169,3 +169,13 @@ https://cbcrclinear-tor.akamaized.net/hls/live/2042765/CBCRCLINEAR_TOR_11/master
 https://cbcrclinear-tor.akamaized.net/hls/live/2042756/CBCRCLINEAR_TOR_2/master5.m3u8
 #EXTINF:-1 tvg-id="CBRTDT.ca",CBC Calgary (CBRT-DT) [Geo-blocked]
 https://cbcrclinear-tor.akamaized.net/hls/live/2042763/CBCRCLINEAR_TOR_9/master5.m3u8
+#EXTINF:-1 tvg-id="ZoomerRadio.ca" status="online",ZoomerRadio
+https://ythls.onrender.com/channel/UCon2a9opDXmI-i3Be6PYBQA.m3u8
+#EXTINF:-1 tvg-id="ClassicalFM.ca" status="online",ClassicalFM
+https://ythls.onrender.com/channel/UC_DweJsPH4mvcRU3aFwVtqg.m3u8
+#EXTINF:-1 tvg-id="TVOKids.ca" status="online",TVO kids
+https://ythls.onrender.com/channel/UCGPPdIn67ZCLhhHABoQpNUw.m3u8
```
Comment (BellezaEmporium) [inline streams/ca.m3u:177]: Seems to either be Geo or not 24/7

**Comment by BellezaEmporium** [comment]
i mean, not in that way. geo in the way that you can't use ythls for this stream, as it won't work.

**Comment by Sphinxroot** [comment]
ClassicalFM do work here :-s

File: streams/ca.m3u (hunk 1 above)
Comment (CoolCanuck) [reply streams/ca.m3u]: Just tested and TVO Kids is indeed Geo to Canada.

File: streams/ca.m3u [hunk 2]
```diff
@@ -173,3 +173,71 @@ https://uni6rtmp.tulix.tv/vbstv/vbsabr.smil/playlist.m3u8
...
+https://flowcaster1-ch.cdn.clearcable.net/C14/smil:C14.smil/playlist.m3u8?iimae0UC2aendtime=1665196120&iimae0UC2astarttime=1665181720&iimae0UC2ahash=NzcDbuotaJAVf1DsNxuGHBlfCXSvHDIt5sqMUMgrRMU=
+#EXTINF:-1 tvg-id="CTBTV.ca",Corporation de Télévision Brandon (CTB TV)
+https://fl001.elpcinc.com/CTB-TV/index.m3u8
+#EXTINF:-1 tvg-id="TCFTV.ca",TCF TV
+http://66.11.33.44:8081/live/TCF/playlist.m3u8
+#EXTINF:-1 tvg-id="CanaldelAssemblee.ca",Assemblée Nationale du Québec
+https://cdn3.wowza.com/5/SVEySlNEQ0FOWXlS/diffusion/canal17/playlist.m3u8
+#EXTINF:-1 tvg-id="CBXFTDT.ca",Ici Radio-Canada Télé Alberta (CBXFT-DT) [Geo-blocked]
+https://rcavlive.akamaized.net/hls/live/704020/cancbxft/master.m3u8
+#EXTINF:-1 tvg-id="CBUFTDT.ca",Ici Radio-Canada Télé Colombie-Britannique - Yukon (CBUFT-DT) [Geo-blocked]
```
Comment (freearhey) [inline streams/ca.m3u:186]:
```suggestion
#EXTINF:-1 tvg-id="CBUFTDT.ca",Ici Radio-Canada Télé Colombie-Britannique/Yukon (CBUFT-DT) [Geo-blocked]
```

File: streams/ca.m3u [hunk 3]
```diff
@@ -173,3 +173,71 @@ https://uni6rtmp.tulix.tv/vbstv/vbsabr.smil/playlist.m3u8
...
+https://rcavlive.akamaized.net/hls/live/704017/cancbuft/master.m3u8
+#EXTINF:-1 tvg-id="CJBRDT.ca",Ici Radio-Canada Télé Est du Québec (CJBR-DT) [Geo-blocked]
+https://rcavlive.akamaized.net/hls/live/704024/cancjbr/master.m3u8
+#EXTINF:-1 tvg-id="CKSHDT.ca",Ici Radio-Canada Télé Estrie (CKSH-DT) [Geo-blocked]
+https://rcavlive.akamaized.net/hls/live/704016/cancksh/master.m3u8
+#EXTINF:-1 tvg-id="CBWFTDT.ca",Ici Radio-Canada Télé Manitoba (CBWFT-DT) [Geo-blocked]
+https://rcavlive.akamaized.net/hls/live/704018/cancbwft/master.m3u8
+#EXTINF:-1 tvg-id="CKTMDT.ca",Ici Radio-Canada Télé Mauricie - Centre-du-Québec (CKTM-DT) [Geo-blocked]
```
Comment (freearhey) [inline streams/ca.m3u:194]:
```suggestion
#EXTINF:-1 tvg-id="CKTMDT.ca",Ici Radio-Canada Télé Mauricie/Centre-du-Québec (CKTM-DT) [Geo-blocked]
```

File: streams/ca.m3u [hunk 4]
```diff
@@ -173,3 +173,71 @@ https://uni6rtmp.tulix.tv/vbstv/vbsabr.smil/playlist.m3u8
...
+https://rcavlive.akamaized.net/hls/live/704021/cancktm/master.m3u8
+#EXTINF:-1 tvg-id="CBLFTDT.ca",Ici Radio-Canada Télé Ontario (CBLFT-DT) [Geo-blocked]
+https://rcavlive.akamaized.net/hls/live/704022/cancblft/master.m3u8
+#EXTINF:-1 tvg-id="CBOFTDT.ca",Ici Radio-Canada Télé Ottawa - Gatineau (CBOFT-DT) [Geo-blocked]
```
Comment (freearhey) [inline streams/ca.m3u:198]:
```suggestion
#EXTINF:-1 tvg-id="CBOFTDT.ca",Ici Radio-Canada Télé Ottawa/Gatineau (CBOFT-DT) [Geo-blocked]
```

File: streams/ca.m3u [hunk 5]
```diff
@@ -173,3 +173,71 @@ https://uni6rtmp.tulix.tv/vbstv/vbsabr.smil/playlist.m3u8
...
+https://cbcrclinear-tor.akamaized.net/hls/live/2042765/CBCRCLINEAR_TOR_11/master5.m3u8
+#EXTINF:-1 tvg-id="CBCTDT.ca",CBC Prince Edward Island (CBCT-DT) [Geo-blocked]
+https://cbcrclinear-tor.akamaized.net/hls/live/2042756/CBCRCLINEAR_TOR_2/master5.m3u8
+#EXTINF:-1 tvg-id="CBRTDT.ca",CBC Calgary (CBRT-DT) [Geo-blocked]
+https://cbcrclinear-tor.akamaized.net/hls/live/2042763/CBCRCLINEAR_TOR_9/master5.m3u8
+#EXTINF:-1 tvg-id="ZoomerRadio.ca" status="online",ZoomerRadio
+https://ythls.onrender.com/channel/UCon2a9opDXmI-i3Be6PYBQA.m3u8
+#EXTINF:-1 tvg-id="ClassicalFM.ca" status="online",ClassicalFM [Geo-blocked]
+https://ythls.onrender.com/channel/UC_DweJsPH4mvcRU3aFwVtqg.m3u8
+#EXTINF:-1 tvg-id="CNV.ca" status="online",CNV - Montreal
```
Comment (freearhey) [inline streams/ca.m3u:240]:
```suggestion
#EXTINF:-1 tvg-id="CNV.ca" status="online",CNV/Montreal
```

File: streams/ca.m3u [hunk 6]
```diff
@@ -173,3 +173,71 @@ https://uni6rtmp.tulix.tv/vbstv/vbsabr.smil/playlist.m3u8
...
+https://media1.radioservers.biz:1936/cnv/cnv/playlist.m3u8
+#EXTINF:-1 tvg-id="NowTV.ca" status="online",CKNO-FM - Now TV 102.3FM - Edmonton
```
Comment (freearhey) [inline streams/ca.m3u:242]:
```suggestion
#EXTINF:-1 tvg-id="NowTV.ca" status="online",Now TV 102.3FM Edmonton (CKNO-FM)
```

**Review by freearhey — CHANGES_REQUESTED** [review]
Please do not use " - " in the title, it may cause a bug in some applications:

image

https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#stream-description-scheme

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**